ansible/roles/girder/tasks/main.yml

```diff
@@ -45,50 +45,6 @@
         - login: cumulus
           type: member
 
-- name: Enable cumulus plugins
-  girder:
-    host: "{{girder_host}}"
-    port: "{{girder_port}}"
-    scheme: "{{girder_scheme}}"
-    apiRoot: "{{girder_api_root}}"
-    username: "hpccloud"
-    password: "{{ hpccloud_password }}"
-    plugins:
-      - cumulus
-      - sftp
-      - taskflow
-    state: present
-
-- name: Enable HC plugins
-  girder:
-    host: "{{girder_host}}"
-    port: "{{girder_port}}"
-    scheme: "{{girder_scheme}}"
-    apiRoot: "{{girder_api_root}}"
-    username: "hpccloud"
-    password: "{{ hpccloud_password }}"
-    plugins:
-      - hpccloud
-      - pvwproxy
-    state: present
-
-- name: restart girder
-  girder:
-    host: "{{girder_host}}"
-    port: "{{girder_port}}"
-    scheme: "{{girder_scheme}}"
-    apiRoot: "{{girder_api_root}}"
-    username: "hpccloud"
-    password: "{{ hpccloud_password }}"
-    put:
-      path: "system/restart"
-
-- name: Wait until server restarts
-  wait_for:
-    host="{{girder_host}}"
-    port="{{girder_port}}"
-    delay=5
-
```

Comment:
I don't know what the migration entailed exactly, but are all of these tasks really no longer used?

Reply:
y, now all plugins are done using 'pip install', and there's no further registration or anything needed. Zach confirmed I could rip them out.